dataactcore/utils/jobQueue.py

```diff
@@ -34,7 +34,7 @@ def enqueue(jobID):
             response = requests.post(url, params)
             return response.json()
 
-        @self.jobQueue.task(name='jobQueue.generate_d1')
+        @self.jobQueue.task(name='jobQueue.generate_d_file')
```

Comment:
correct, which is specified by `d_file_id` which is in the d_file_metadata db table